PyMPDATA/solver.py

```diff
@@ -107,8 +113,13 @@ def advector(self) -> VectorField:
 
     @property
     def g_factor(self) -> ScalarField:
-        """g_factor field (with halo), unmodified by advance(),
-        assumed to be constant-in-time"""
+        """G_factor field (with halo), unmodified by advance(), assumed to be constant-in-time.
+        Can be used as a Jacobian for coordinate transformations,
+        e.g. into spherical coordinates.
+        For this type of usage, see
+        Williamson_and_Rasch_1989_as_in_Jaruga_et_al_2015_Fig_14 in PyMPDATA-examples.
```

Comment:
we can use clickable links in docstrings to refer to other API elements, and pdoc CI runs will fail if these are broken links